Sources/AWSSDKSwiftCore/Message/AWSResponse+HAL.swift

```diff
@@ -17,31 +17,36 @@ import NIO
 
 // AWS HAL services I know of are APIGateway, Pinpoint, Greengrass
 extension AWSResponse {
-    /// process hal+json date. Extract properties from HAL
-    func getHypertextApplicationLanguageBody() throws -> Body {
-        guard case .json(let buffer) = self.body,
+    /// return if body is hypertext application language
+    /// - Returns: <#description#>
+    func IsHypertextApplicationLanguage() -> Bool {
```

Comment:
This is more a readonly `var`, isn't it?

Reply:
Ok can change.

Reply:
done